pb/protos/commands.proto

```diff
@@ -2000,10 +2000,23 @@ message Rpc {
                         UNKNOWN_ERROR = 1;
                         BAD_INPUT = 2;
                         INTERNAL_ERROR = 3;
-                        NO_OBJECTS_TO_IMPORT = 5;
-                        IMPORT_IS_CANCELED = 6;
-                        LIMIT_OF_ROWS_OR_RELATIONS_EXCEEDED = 7;
                         FILE_LOAD_ERROR = 8;
+                        IMPORT_IS_CANCELED = 6;
```

Comment:
Why are errors duplicated both in pb/protos and pkg/lib/pb/model/protos?

Reply:
Because pb error code are used in service command, model error codes are used in Notification. For now I don't know, how to unify them, because it requires to overwrite generation of pb files for clients.